src/app/pages/activation/activation-routing.module.ts

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
Like you said: try to figure out a way so that you don't have to use inidvidual dummy ng routing modules for every single page. Of course, you can still merge this as this can still be implemented in the future.